.travis.yml

```diff
@@ -16,7 +16,7 @@ before_install:
 # Install packages
 install:
   - conda install --yes python=$TRAVIS_PYTHON_VERSION pandas nose sqlalchemy
-  - conda install --yes -c ooi ion-functions
+  - conda install --yes -c ooi ion-functions=2.4.3
```

Comment:
probably not a good idea to lock the ion-functions to a particular version. Shouldn't we be using the latest, or have some process to keep the ion-functions consistent across all packages?

Reply:
I have reverted the change in the travis.yml file.

Reply:
How about specifying a minimum version? 
`- conda install --yes -c ooi ion-functions>=2.4.3`

Reply:
It looks like specifying numpy=1.12 and pandas=0.19.2 (same as stream engine) did the trick.